### PR TITLE
no more HTML saving! It's so 80s

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
Stop saving HTML that no one uses. It takes 5mb per file for each build

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/logging/Log.java [hunk 1]
```diff
@@ -55,19 +55,12 @@ public static void clearLogStack() {
...
   public static void log(String command, String description, boolean success, WebDriver driver) {
     LOGS_RESULTS.add(success);
     imageCounter += 1;
 
     try {
       new Shooter().savePageScreenshot(Log.SCREEN_PATH + Log.imageCounter, driver);
-      CommonUtils.appendTextToFile(Log.SCREEN_PATH + Log.imageCounter + ".html",
-                                   Log.getPageSource(driver));
       VelocityWrapper.fillErrorLogRow(Arrays.asList(LogLevel.ERROR), description, Log.imageCounter);
     } catch (Exception e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/logging/Log.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/common/logging/Log.java [hunk 2]
```diff
@@ -248,8 +239,6 @@ public static void logAssertionStacktrace(AssertionError exception) {
       String html = VelocityWrapper.fillErrorLogRow(classList, exceptionMessage, Log.imageCounter);
       try {
         new Shooter().savePageScreenshot(Log.SCREEN_PATH + Log.imageCounter, driver);
-        CommonUtils.appendTextToFile(Log.SCREEN_PATH + Log.imageCounter + ".html",
-                                     Log.getPageSource(driver));
         CommonUtils.appendTextToFile(Log.LOG_PATH, html);
       } catch (Exception e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/logging/Log.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 6 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Log.java#L39](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L39): Add a private constructor to hide the implicit public one. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)
1. ![MAJOR][MAJOR] [Log.java#L122](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L122): Remove this unused "escapedDescription" local variable. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)
1. ![MAJOR][MAJOR] [Log.java#L180](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L180): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [Log.java#L312](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L312): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [Log.java#L313](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L313): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MINOR][MINOR] [Log.java#L272](https://github.com/Wikia/selenium-tests/blob/f4624e78d031c39f51753b1e69212244fb807280/src/test/java/com/wikia/webdriver/common/logging/Log.java#L272): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS134)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/logging/Log.java (hunk 1 above)
Comment (nandy-andy) [reply src/test/java/com/wikia/webdriver/common/logging/Log.java]: @ludwikkazmierczak decide on one, please ;)

File: src/test/java/com/wikia/webdriver/common/logging/Log.java (hunk 2 above)
Comment (nandy-andy) [reply src/test/java/com/wikia/webdriver/common/logging/Log.java]: Ditto :)

**Pull request merged**